DOCUMENTATION.md

```diff
@@ -286,4 +286,23 @@ content-cli push asset -p my-profile-name -f asset_73d39112-73ae-4bbe-8051-3c0f1
 ```
 
 | Note: Pushing analysis from Process Analytics to Studio will only work if you have used the ***--asset*** option when pulling. |
+
+### Pull and Push Analysis Bookmarks in Studio
+
+Enable users to pull and push bookmarks using content-cli. For pulling analysis bookmarks
+you can specify --type (shared/all), and by default it fetches user bookmarks.
+
+```
+// Pull analysis bookmarks
+content-cli pull bookmarks --profile my-profile-name --id 73d39112-73ae-4bbe-8051-3c0f14e065ec --type shared
```

Comment:
I think having this command use the asset key (`rootKey.key`) instead of the asset id would be more user-friendly also consistent with other commands, as the asset id is hidden from UI and only attainable from the URL. I'm also fine with leaving it like this, however please consider this as it will most definitely cause confusion.